roles/couchbase-infra/tasks/main.yml

```diff
@@ -0,0 +1,209 @@
+---
+# Install Couchbase Operator (Assume rook is already in place? or should we check?)
+- name: Create CRD for Couchbase Operator
+  k8s:
+    definition: "{{ lookup('file', 'crd.yaml') }}"
+  when: couchbase.servers.size > 0
+
+- name: Reticulating Splines
+  k8s:
+    definition:
+      apiVersion: v1
+      kind: Namespace
+      metadata:
+        # This is the namespace coded into the operator
+        # from operatorhub.io. Best just to embrace it
+        # rather than fight it.
+        name: my-couchbase-enterprise
+  when: couchbase.servers.size > 0
+
+- name: Create Cluster Role for Service Account
+  k8s:
+    definition: "{{ lookup('file', 'cluster-role-sa.yaml') }}"
+  when: couchbase.servers.size > 0
+
+- name: Create Service Account
+  k8s:
+    definition:
+      apiVersion: v1
+      kind: ServiceAccount
+      metadata:
+        name: couchbase-operator
+        #namespace: '{{ meta.namespace }}'
+        namespace: my-couchbase-enterprise
```

Comment:
maybe we just var this out and put it in the CR (so we can describe it in one place for people to understand)

Reply:
Ack; will change